deps.versions.toml

```diff
@@ -1,25 +1,25 @@
 [versions]
 
 decompose = "2.0.1-compose-experimental"
-kotlin = "1.8.20"
+kotlin = "1.8.21"
```

Comment:
Is this required or just on-the-fly?

Reply:
I looked up the beta01 dependencies to check if this is actually required (I just did it on-the-fly before). It says:

> This version of Compose Multiplatform is based on the next Jetpack Compose libraries:
> [Compiler 1.5.0](https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.0)
> [Runtime 1.5.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.5.0-beta03)
> [UI 1.5.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.5.0-beta03)
> [Foundation 1.5.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.5.0-beta03)
> [Material 1.5.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-material#1.5.0-beta03)
> [Material3 1.1.1](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.1.1)

Kotlin was updated to 1.9.0 in Compose Compiler 1.5.0 (according to the [docs](https://developer.android.com/jetpack/androidx/releases/compose-kotlin)), so I guess it should probably be upgraded all together?

_Note: Upgrading Kotlin to 1.9.0 would require updates in the `gradle-setup-plugin`_